apps/src/templates/TeacherFeedback.jsx

```diff
@@ -39,7 +51,22 @@ class TeacherFeedback extends Component {
     return (
       <div style={styles.container}>
         <div style={styles.header}>{i18n.forTeachersOnly()}</div>
-        <div style={styles.content}>Coming soon: You’ll be able to use this tab to give feedback to your students about their work</div>
+        {!this.props.internalFlag &&
+          <div style={styles.content}>Coming soon: You’ll be able to use this tab to give feedback to your students about
+            their work
+          </div>
```

Comment:
Formatting: You can start the content on the next line.  React (unlike HTML) will handle the whitespace correctly.